proto/README.md

```diff
@@ -24,6 +25,11 @@ something happening to a Cloud Storage object uses `StorageObjectData`
 as the data message. Each envelope message specifies the CloudEvent
 type it's associated with via a proto annotation.
 
+Initially we will generate libraries that only contain the data
+messages. The event messages serve as useful documentation for the
+event type strings and extension attributes. They will be included
+in the libraries at a later date if that proves useful.
```

Comment:
I generally don't include "we will" statements in READMEs, and rather add TODOs and link to GitHub issues.

But this is fine.